### PR TITLE
Aaron test (please ignore)

### PR DESCRIPTION
**Pull request by vontell**
This PR does stuff

---

Find the pull request instructions [here](https://www.notion.so/regressiongg/Pull-Request-Process-0d57a7eb582a446983edfacafa406f1e?pvs=4)

Every reviewer and the owner of the PR should consider these points in their request (feel free to copy this checklist so you can fill it out yourself in the overall PR comment)

- [ ] The code is extensible and backward compatible
- [ ] New public interfaces are extensible and open to backward compatibility in the future
- [ ] If preparing to remove a field in the future (i.e. this PR removes an argument), the argument stays but is no longer functional, and attaches a deprecation warning. A linear task is also created to track this deletion task.
- [ ] Non-critical or potentially modifiable arguments are optional
- [ ] Breaking changes and the approach to handling them have been verified with the team (in the Linear task, design doc, or PR itself)
- [ ] The code is easy to read
- [ ] Unit tests are added for expected and edge cases
- [ ] Integration tests are added for expected and edge cases
- [ ] Functions and classes are documented
- [ ] Migrations for both up and down operations are completed
- [ ] A documentation PR is created and being reviewed for anything in this PR that requires knowledge to use
- [ ] Implications on other dependent code (i.e. sample games and sample bots) is considered, mentioned, and properly handled
- [ ] Style changes and other non-blocking changes are marked as non-blocking from reviewers

### CODE REVIEW

File: Runtime/Scripts/RGServiceManager.cs [hunk 1]
```diff
@@ -512,6 +512,37 @@ public async Task UploadValidations(long botInstanceId, string filePath, Action
...
+                    {
+                        RGDebug.LogDebug(
+                            $"RGService UploadLogs response received");
+                        onSuccess.Invoke();
+                    },
+                    onFailure: async (f) =>
+                    {
+                        RGDebug.LogWarning($"Failed to upload logs for bot instance {botInstanceId}: {f}");
+                        onFailure.Invoke();
+                    }
```
[the remark this answers is not in the document]
Comment (analogrelay) [reply Runtime/Scripts/RGServiceManager.cs:537]: Interestingly, I did notice this issue (it occurs with our existing validations upload). Fortunately, if the file is missing, it produces a warning rather than throwing :).
